plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -328,6 +329,12 @@ public void initialize(URI uri, Configuration conf)
                 .withUserAgentPrefix(userAgentPrefix)
                 .withUserAgentSuffix("Trino");
 
+        String connectTtlValue = conf.get(S3_CONNECT_TTL);
+        if (!isNullOrEmpty(connectTtlValue)) {
+            Duration connectTtl = Duration.valueOf(connectTtlValue);
+            configuration.setConnectionTTL(toIntExact(connectTtl.toMillis()));
```

Comment:
`setConnectionTTL()` accepts long and `toMillis()` returns long. Why do we enforce int type with `toIntExact`? Same for `TrinoS3ClientFactory`.

Reply:
> `setConnectionTTL()` accepts long and `toMillis()` returns long. Why do we enforce int type with `toIntExact`? Same for `TrinoS3ClientFactory`.

Sorry for the mistake, it should accepts long type, i have modified this, thanks